### PR TITLE
added some information about the unit tests

### PR DESCRIPTION
**Pull request by fretn**
This is useful for people who don't want to read the full unittest documentation but still want to run the unit test example

### CODE REVIEW

File: docs/Installation.rst [hunk 1]
```diff
@@ -135,9 +135,10 @@ Running unit tests
 After completion of the bootstrap procedure and loading the
 ``EasyBuild`` module, try running the EasyBuild unit tests::
 
+    export TEST_EASYBUILD_MODULES_TOOL=Lmod
```
Comment (boegel) [inline docs/Installation.rst]: maybe add a comment above:

```shell
# specify modules tool to use: EnvironmentModulesC (default), EnvironmentModulesTcl, Lmod)
# see also http://easybuild.readthedocs.org/en/latest/Configuration.html#modules-tool-modules-tool
export TEST_EASYBUILD_MODULES_TOOL=Lmod
# run full unit test suite for EasyBuild framework
python -m test.framework.suite
```

**Comment by boegel** [comment]
@fretn: please drop in a comment if you're tackled the remark, would be great to merge this in

**Comment by fretn** [comment]
@boegel: done

**Comment by boegel** [comment]
@fretn: excellent, thanks

next time, please target the `develop` branch when issuing a PR (I'll make sure this change is also synced in `develop`)

for documentation, it's less of a problem, but for code changes, PR *must* go to `develop`, we only update `master` for a new EB release

Congratulations with your first pull request to EasyBuild! :+1:

**Pull request merged**